controllers/openshift_controller.go

```diff
@@ -187,6 +188,30 @@ func (r *KataConfigOpenShiftReconciler) processDaemonsetForCR(operation DaemonOp
 								},
 							},
 							Env: []corev1.EnvVar{
+								{
+									Name: "PAYLOAD_SECRET_USERNAME_ENV_VAR",
```

Comment:
Is `PAYLOAD_REGSITRY_USERNAME` and `PAYLOAD_REGSITRY_PASSWORD` a better name ?

Reply:
Yes, it's shorter and precise. I'll change it. Thanks!